examples/with-sentry/pages/_error.js

```diff
@@ -1,56 +1,82 @@
-import NextErrorComponent from 'next/error'
+/**
```

Comment:
@lobsterkatie, can we please remove most (if not all) comments from `_error.js`?
As a user of next.js, I am not interested in bits and pieces of hows and whys, I just want this to work. If you feel that that information is crucial for the usage (not setup), I would move it into [README.md](https://github.com/vercel/next.js/blob/canary/examples/with-sentry/README.md) or our docs.

Reply:
See below.